.evergreen/config.yml

```diff
@@ -172,101 +172,122 @@ functions:
 
 pre:
   - func: "fetch source"
-  - func: "prepare resources"
   - func: "install dependencies"
 
-post:
```

Comment:
similarly, both of these tasks are irrelevant for atlas tests. we haven't started orchestration, and cleanup just removes the drivers evg tools we haven't downloaded.